rules-tests/AnnotationsToAttributes/Rector/ClassMethod/DataProviderAnnotationToAttributeRector/Fixture/already_in_use.php.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rector\PHPUnit\Tests\AnnotationsToAttributes\Rector\ClassMethod\DataProviderAnnotationToAttributeRector\Fixture;
+
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\TestCase;
+
+final class AlreadyInUse extends TestCase
+{
+    /**
+    * @dataProvider someMethod()
+    */
+    public function test(): void
+    {
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\PHPUnit\Tests\AnnotationsToAttributes\Rector\ClassMethod\DataProviderAnnotationToAttributeRector\Fixture;
+
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\TestCase;
+
+final class AlreadyInUse extends TestCase
+{
+    #[\PHPUnit\Framework\Attributes\DataProvider('@dataProvider')]
```

Comment:
I think there should be a method name instead:

```suggestion
    #[\PHPUnit\Framework\Attributes\DataProvider('someMethod')]
```

Reply:
Oops, I will check :)

Reply:
https://github.com/rectorphp/rector-phpunit/pull/524